upload-tarballs-to-oasis

```diff
@@ -175,7 +175,7 @@ die_if_grid_commands_not_in_path() {
         set -o errexit
         which grid-proxy-info
         which grid-proxy-init
-        which gsissh
+        which ssh
```

Comment:
You should drop this function entirely if you're not using GSI auth anymore.

Reply:
also drop `get_proxy_if_needed_or_die`

Reply:
I did as you requested. Do we need to rerun the tests? It looks like the EPEL repodata is out of date again. I think that I should be able to merge, because my changes aren't tested anyway. (BTW, I tested them with today's release.)

Reply:
Yeah I think we can merge.